docs/functions.rst

```diff
@@ -0,0 +1,50 @@
+Useful Functions
+================
+There are some functions that interact with multiple types of objects to apply
+calibrations solutions and flagging to other objects.
+
+.. autofunction:: pyuvdata.uvcalibrate
+
+.. autofunction:: pyuvdata.apply_uvflag
+
+
+Utility Functions
```

Comment:
I would move up this subsection to group all the functions in one list, maybe add a `... warning::` block to highlight the change in namespace.